azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/FunctionApp.java

```diff
@@ -136,11 +138,18 @@
          */
         interface Blank extends DefinitionWithRegion<NewAppServicePlanWithGroup> {
             /**
-             * Uses an existing app service plan for the function app.
+             * Uses an existing Windows app service plan for the function app.
              * @param appServicePlan the existing app service plan
              * @return the next stage of the definition
              */
             ExistingAppServicePlanWithGroup withExistingAppServicePlan(AppServicePlan appServicePlan);
```

Comment:
The naming should be consistent.

Reply:
You mean function naming or docstring naming?

Reply:
function naming. Second though, we need back compatible with previous API, so that the change might not necessary. But anyway, user can be confused in the case.